.circleci/config.templ.yml

```
@@ -186,6 +186,7 @@ commands:
                 environment:
                   DD_TRACE_AGENT_URL: << parameters.trace_agent_url >>
                   RIOT_RUN_RECOMPILE_REQS: "<< pipeline.parameters.riot_run_latest >>"
+                  DD_CIVISIBILITY_AGENTLESS_ENABLED: "true"
```

If I'm not mistaken, yaml is string by default, no?
```suggestion
                  DD_CIVISIBILITY_AGENTLESS_ENABLED: true
```

---

It is but I like my mistakes sprinkled throughout my PRs.